config/config.default.js

```diff
@@ -63,6 +63,10 @@ var path = require('path'),
             defaultConnectionRouter: 'basic3' //'basic', 'basic2', 'basic3'
         },
 
+        core: {
```

Comment:
My anticipation is that there will be more options for the core and not for the constraints.So adding a constraints group will clutter the config IMO.
Since the constraints are defined on core-nodes and the code for checking constraints is in common/core/users - core is not that awful.